src/app/forms/templates/search/single-search-result.template.ts

```diff
@@ -1,15 +1,15 @@
 import { ValidatorNames } from '@forms/models/validators.enum';
 import { FormNode, FormNodeTypes, FormNodeViewTypes } from '../../services/dynamic-form.types';
 
-export function createSingleSearchResult(systemNumber?: string): FormNode {
+export function createSingleSearchResult(systemNumber?: string, vin?: string): FormNode {
```

Comment:
Should the parameters `systemNumber` and `vin` not be mandatory here?

If a call would be made to this function with both set to `null`, the URL would be "/tech-records//" and that would crash, right?

Reply:
This is already an issue if you don't pass it a systemNumber which then reroutes you back to the home page.